cli/cmd/install_test.go

```diff
@@ -174,20 +174,51 @@ func TestValidate(t *testing.T) {
 		}
 	})
 
-	t.Run("Rejects invalid log level", func(t *testing.T) {
+	t.Run("Rejects invalid controller log level", func(t *testing.T) {
 		options := newInstallOptions()
 		options.controllerLogLevel = "super"
 		expected := "--controller-log-level must be one of: panic, fatal, error, warn, info, debug"
 
 		err := options.validate()
 		if err == nil {
-			t.Fatalf("Expected error, got nothing")
+			t.Fatal("Expected error, got nothing")
 		}
 		if err.Error() != expected {
 			t.Fatalf("Expected error string\"%s\", got \"%s\"", expected, err)
 		}
 	})
 
+	t.Run("Properly validates proxy log level", func(t *testing.T) {
+		testCases := []struct {
+			input string
+			valid bool
+		}{
+			{"", false},
+			{"info", true},
+			{"somemodule", true},
+			{"linkerd2_proxy=debug", true},
+			{"linkerd2_proxy=foobar", false},
+			{"linker2d_proxy,std::option", true},
+			{"warn,linkerd2_proxy=info", true},
+			{"warn,linkerd2_proxy=foobar", false},
```

Comment:
It would be nice to see some test cases in here with characters that are never valid in a proxy log level string.

Reply:
will do :+1: